frontend/workflows/projectCatalog/src/details/info/repositoryRow.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Grid } from "@clutch-sh/core";
+import type { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { faCode, faCodeBranch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { LinkText } from "../helpers";
+
+interface ProjectRequests {
```

Comment:
This is more for later but I think we could improve this name to be a bit more descriptive. I think it's ok to include something like pull requests in this name as it's more descriptive

Reply:
Modified to be `ProjectPullRequests`, I had it as `ProjectRequests` originally since I was leaving it up to the user to specify the type, but I've removed that logic anyways.